test/e2e/specs/site-editor/template-revert.spec.js

```diff
@@ -279,11 +291,25 @@ class TemplateRevertUtils {
 	}
 
 	async revertTemplate() {
-		await this.page.click( 'role=button[name="Show template details"i]' );
+		await this.page.click( 'role=button[name="Settings"i]' );
+		const isTemplateTabVisible = await this.page
+			.locator(
+				'role=region[name="Editor settings"i] >> role=button[name="Template"i]'
+			)
+			.isVisible();
+		if ( isTemplateTabVisible ) {
+			await this.page.click(
+				'role=region[name="Editor settings"i] >> role=button[name="Template"i]'
+			);
+		}
+		await this.page.click(
```

Comment:
Seems like this test is flaky for some reason (you can see the error in the "Files changed" tab). Do you have any idea why? Could it be a legit failure or the test is not deterministic?

Reply:
Oh, nvm. I see that @Mamaduka has linked https://github.com/WordPress/gutenberg/pull/50851 to this so probably this is already fixed 😅 .